### PR TITLE
gh-119786: Replace a Sphinx role with literal syntax

### PR DESCRIPTION
**Pull request by AA-Turner**
* Issue: gh-119786

### CODE REVIEW

File: InternalDocs/exception_handling.md [hunk 1]
```diff
@@ -190,5 +190,5 @@ Exception Chaining Implementation
 [Exception chaining](https://docs.python.org/dev/tutorial/errors.html#exception-chaining)
 refers to setting the ``__context__`` and ``__cause__`` fields of an exception as it is
 being raised. The ``__context__`` field is set by ``_PyErr_SetObject()`` in
-:cpy-file:`Python/errors.c` (which is ultimately called by all ``PyErr_Set*()`` functions).
+``Python/errors.c`` (which is ultimately called by all ``PyErr_Set*()`` functions).
```
Comment (iritkatriel) [inline InternalDocs/exception_handling.md]: We could link to the file like, e.g., on line 184.

Comment (AA-Turner) [reply InternalDocs/exception_handling.md]:
```suggestion
[Python/errors.c](https://github.com/python/cpython/blob/main/Python/errors.c)
(which is ultimately called by all ``PyErr_Set*()`` functions).
```

**Review by picnixz — APPROVED** [review]
I confirmed the link!

**Review by iritkatriel — APPROVED** [review]
Thanks.

**Pull request merged**